### PR TITLE
Add error message on server error, re enable the save button

### LINKED ISSUE #2098: Don't keep the button save desable after server error
Permalink directly where we can reproduce the issue: https://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop_alt/?lang=fr&map_y=170087&map_x=639795&baselayer_ref=asitvd.fond_gris&map_zoom=3&tree_group_layers_Transport=fuel&tree_group_layers_Edit=line%2Cpolygon%2Cpoint&tree_group_layers_Snapping&tree_group_layers_Snapping%20tollerance&tree_group_layers_Snapping%20no%20edge&tree_group_layers_Snapping%20no%20vertex&tree_groups=Edit%2CSnapping%2CSnapping%20tollerance%2CSnapping%20no%20edge%2CSnapping%20no%20vertex

Steps to reproduce the issue:

* Login
* Tool: Editing
* Layer: Polygon
* Button without text !!!
* Draw a polygon
* Insert a name
* Save

Result:
The button is disable after server error

Expected result:
* Having an error message
* The button should be enable o be able to try it again.

The issue concern the following environment:

* Firefox 50 Ubuntu Xenial

The following environment is working as expected:

* Neiser

### PR DESCRIPTION
**Pull request by sbrunner**
fix #2098

### CODE REVIEW

File: contribs/gmf/src/directives/editfeature.js [hunk 1]
```diff
@@ -526,24 +532,27 @@ gmf.EditfeatureController.prototype.save = function() {
...
+    ) :
     this.editFeatureService_.insertFeatures(
       this.editableNode_.id,
       [feature]
-    ).then(
-      this.handleEditFeature_.bind(this)
     );
-  }
+  promise.then(
+    function(responce) {
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/editfeature.js]: responce -> response (and below)

**Comment by sbrunner** [comment]
Not sure to understand your comment ...

**Comment by ger-benjamin** [comment]
I just ask me about the whole workflow in this file. That's not a part of your changes.

In this file, there is a `gmf.EditfeatureController.prototype.cancel` function. But this function don't set the `this.pending` value to false. So... even if we cancel the request, the state is always and forever to `this.pending = true` ?

**Comment by sbrunner** [comment]
No because it's the pending of the request deferent to the dirty :-)

**Pull request merged**